server/src/test/integration/org/apache/cassandra/sidecar/testing/IntegrationTestModule.java

```diff
@@ -47,13 +58,26 @@
  */
 public class IntegrationTestModule extends AbstractModule
 {
+    public static final String ADMIN_IDENTITY = "spiffe://cassandra/sidecar/admin";
```

Comment:
nit: same string declared in multiple locations. maybe consolidate.

Reply:
It is declared once in `IntegrationTestModule` and once in `MutualTlsAuthenticationHandlerTest`. Since both are different packages, integration test and unit test, there is no repetition.